sklearn/metrics/tests/test_common.py

```diff
@@ -1110,3 +1110,101 @@ def test_no_averaging_labels():
             score_labels = metric(y_true, y_pred, labels=labels, average=None)
             score = metric(y_true, y_pred, average=None)
             assert_array_equal(score_labels, score[inverse_labels])
+
+
+def test_metric_permutation_invariance_multiclass():
+    rng = np.random.RandomState(42)
```

Comment:
Please add a comment that you're drawing classes from a random multinomial. Although I'm not sure this is necessary, when [0] * 20 + [1] * 10 + [2] * 5 + [3] * 6 + [4] * 7 or something would do.